app/Http/Controllers/Store/CentreController.php

```diff
@@ -106,6 +106,10 @@ public function exportRegistrationsSummary(Centre $centre)
      */
     private function getCentreRegistrationsSummary(array $centre_ids, $dateFormats = [], $excludeColumns = [])
     {
+        // Get User
+        /** @var CentreUser $user */
+        $user = Auth::user();
```

Comment:
This works, but may be unnecessary, as the user is checked for the permissions above in `exportRegistrationsSummary`

We can pass "Date file was Downloaded" as a value in the "excludeColumns" array.
Then we can check if we need to add the date to the array further down

Reply:
I've amended my code. However, I was a bit a confused about your remark:
> Then we can check if we need to add the date to the array further down